src/css/skins/vapor.less

```diff
@@ -149,7 +148,8 @@
 
         .jw-option {
             color: #fff;
-
+            
+            &:hover,
```

Comment:
@kasperific so now vapor has the same problem where options are only set for cc menu?